src/transformers/models/beit3/modeling_beit3.py

```diff
@@ -0,0 +1,1384 @@
+            configuration. Check out the [`~PreTrainedModel.from_pretrained`] method to load the model weights.
+"""
+
+BEIT3_MODEL = r"""
+    Args:
+        input_ids (`torch.LongTensor` of shape `({0})`):
+            Indices of input sequence tokens in the vocabulary.
+
+            Indices can be obtained using [`AutoTokenizer`]. See [`PreTrainedTokenizer.encode`] and
+            [`PreTrainedTokenizer.__call__`] for details.
+
+            [What are input IDs?](../glossary#input-ids)
+        pixel_values (`torch.FloatTensor` of shape `(batch_size, num_channels, height, width)`):
+            Pixel values. Pixel values can be obtained using [`AutoImageProcessor`]. See
+            [`BeitImageProcessor.__call__`] for details.
+        text_padding_position (`torch.LongTensor` of shape `({0})`):
+            Padding mask for input tokens , of same shape as `input_ids`
+
+            - 1 indicates the token is **not masked**,
+            - 0 indicates the token is **masked**.
+        attention_mask (`torch.FloatTensor` of shape `({0})`, *optional*):
+            Mask to avoid performing attention on padding token indices. Mask values selected in `[0, 1]`:
+
+            - 1 for tokens that are **not masked**,
+            - 0 for tokens that are **masked**.
+
+            [What are attention masks?](../glossary#attention-mask)
+        vision_masked_position (`torch.LongTensor` of shape `({1})`):
+            Padding mask for input tokens , of same shape as `pixel_values`
+
+            - 1 indicates the token is **not masked**,
+            - 0 indicates the token is **masked**.
+        incremental_state (`Dict`):
+            A Dictionary containing the incremental states layerwise/
+        positions (`int`):
+            Position of where text representations end and image representation start.
+        return_dict (`bool`, *optional*):
+            Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
+        output_hidden_states (`bool`, *optional*):
+            Whether or not to return the hidden states of all layers. See `hidden_states` under returned tensors for
+            more detail.
+"""
+
+BEIT3_FOR_VISUALREASONING_INPUTS_DOCSTRING = r"""
+    Args:
+        input_ids (`torch.LongTensor` of shape `({0})`):
+            Indices of input sequence tokens in the vocabulary.
+
+            Indices can be obtained using [`AutoTokenizer`]. See [`PreTrainedTokenizer.encode`] and
+            [`PreTrainedTokenizer.__call__`] for details.
+
+            [What are input IDs?](../glossary#input-ids)
+        pixel_values (`torch.FloatTensor` of shape `(batch_size, 2 ,num_channels, height, width)`):
+            Pixel values. Pixel values can be obtained using [`AutoImageProcessor`]. See
+            [`BeitImageProcessor.__call__`] for details.
+        padding_mask (`torch.LongTensor` of shape `({0})`):
+            Padding mask for input tokens , of same shape as `input_ids`
```

Comment:
this should then also be called `text_padding_mask`, assuming it only refers to text tokens

Reply:
This has been addressed, we renamed the padding_mask to attention_mask to have uniformity across the repo and to support the pipeline. Sorry I did not reply it earlier.